tools/cli/app_test.go

```diff
@@ -200,6 +200,69 @@ var describeDomainResponseServer = &types.DescribeDomainResponse{
 	},
 }
 
+// This is used to mock the domain deletion confirmation
+func (s *cliAppSuite) mockStdinInput(input string) func() {
+	oldStdin := os.Stdin
+	r, w, _ := os.Pipe()
```

Comment:
I think you should handle an error.
```suggestion
	r, w, err := os.Pipe()
	s.Require().NoError(err)
```